sources/web/datalab/templates/nb.html

```diff
@@ -199,37 +180,16 @@
         </div>
         <div id="sidebarArea" style="display:none">
           <div id="sidebarContent">
-            <div class="btn-group">
-              <button id="toggleSidebarButton" type="button" class="toolbar-btn" title="Toggle the size of the sidebar">
-                <i class="material-icons">chevron_left</i>
-              </button>
+            <div>
+              <div class="btn-group pull-right">
```

Comment:
this looks like it's superimposed on the sidebar content, this is likely because of the `pull-right` class, which floats it. we'll need to auto-overflow it's container div.